tests/vm_test.go

```diff
@@ -730,7 +732,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 						"New VMI was created, but StateChangeRequest was never cleared")
 				})
 
-				It("should not remove a succeeded VMI", func() {
+				It("[test_id:2188] should not remove a succeeded VMI", func() {
```

Comment:
I am missing the action of Connect to VM and run power off and wait for VMI to have vm.status.phase = Succeeded -- the VMI will not be deleted verify VM's runStrategy is still RerunOnFailure

Reply:
Confirmed test is correct in offline conversation with @ILpinto